src/main/java/no/fintlabs/azure/storage/StorageAccountService.java

```diff
@@ -57,6 +58,7 @@ public StorageAccount add(FlaisCrd<? extends AzureSpec> crd, String path, Storag
                 .withTag(TAG_INSTANCE, crd.getMetadata().getLabels().get("app.kubernetes.io/instance"))
                 .withTag(TAG_PART_OF, crd.getMetadata().getLabels().get("app.kubernetes.io/part-of"))
                 .withTag(TAG_CRD_NAME, crd.getMetadata().getName())
+                .withTag(TAG_LIFESPAN_DAYS, String.valueOf(lifespanDays))
```

Comment:
Er det snakk om lifespan på storage accounten, eller lifespan på blobs i storage accounten?

Blob storage støtter lifespan/retention på blobs, men om man ønsker å sette denne til en custom verdi kan ikke dette gjøres gjennom tags. Det må lages en ny lifecycle rule hvor man setter lifespan på blobs.

Tenker det er fint å bevare lifespan tagen på storage accounten slik at det blir lettere for oss å søke, etc...

https://learn.microsoft.com/en-us/azure/storage/blobs/lifecycle-management-policy-configure?tabs=azure-portal